src/utils/api_calls.py

```diff
@@ -1,4 +1,12 @@
 import requests
+import yaml
+import os
+
+# Read YAML file 
+secrets_file_path = os.path.join(os.path.dirname(os.path.dirname(os.path.dirname(__file__))), 'config\secrets.yml')
```

Comment:
os.path.join should take care of adding all the slashes to allow use on different os

Reply:
got it now!